prometheus-ksonnet/lib/prometheus-ha-mixin.libsonnet

```diff
@@ -1,104 +1 @@
-local k = import 'ksonnet-util/kausal.libsonnet';
```

Comment:
suggestion:

```
(import 'prometheus/ha-mixin.libsonnet')
```

There are a few people using this mixin, by name, this will prevent that from failing.

Reply:
Already there, please check again.